src/main/java/graphql/parser/SourceLocationHelper.java

```diff
@@ -0,0 +1,23 @@
+package graphql.parser;
+
+import graphql.language.SourceLocation;
+import org.antlr.v4.runtime.Token;
+
+class SourceLocationHelper {
+
+    static SourceLocation mkSourceLocation(MultiSourceReader multiSourceReader, Token token) {
```

Comment:
I know we have it already at some other places, but I really prefer `create...` over `mk...`. I think it is just clearer.